cishouseholds/derive.py

```diff
@@ -872,3 +872,38 @@ def assign_correct_age_at_date(df: DataFrame, column_name_to_assign, reference_d
         + F.round((F.col("month_more") + F.col("day_more")) / 3, 0).cast("int"),
     )
     return df.drop("month_more", "day_more")
+
```

Comment:
Is it also needed to include the derivation of these into the ETL scripts?
Currently not deriving the two variables.